var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -84,6 +84,34 @@ class Geant4(CMakePackage):
     variant("timemory", default=False, description="Use TiMemory for profiling", when="@9.5:")
     variant("vtk", default=False, description="Enable VTK support", when="@11:")
 
+    # For most users, obtaining the Geant4 data via Spack will be useful; the
+    # sticky, default-enabled `+data` variant ensures that this happens.
+    # Furthermore, if this variant is enabled, Spack will automatically set the
+    # necessary environment variables to ensure that the Geant4 code runs
+    # correctly.
+    #
+    # However, the Geant4 data is also large and it is, on many machines used
+    # in HEP, already available via e.g. CVMFS. In these cases, users can save
+    # network bandwidth by using externally supplied Geant4 data. This can be
+    # done in two different ways.
+    #
+    # The first is to declare the Geant4 data directories as externals. This
+    # can be done by manually adding them to the `packages.yaml` file, e.g.:
+    #
+    # ```
+    # g4radioactivedecay:
+    #   externals:
+    #   - spec: g4radioactivedecay@5.6
+    #     prefix: /cvmfs/geant4.cern.ch/
+    # ```
+    #
+    # Alternatively, the `~data` variant can be supplied; in this case, Spack
+    # will not attempt to use the `geant4-data` spec at all. It is then
+    # essential to set up the `GEANT4_INSTALL_DATADIR` variable manually.
```

Comment:
The `GEANT4_INSTALL_DATADIR` only applies to the _build_ of Geant4, so do we want to explicitly say this means editing the `package.py`? The other option is to note that environment variable(s) can be used to override the expected location, and to point to the post install setup guide in Geant4's Installation Guide: https://geant4-userdoc.web.cern.ch/UsersGuides/InstallationGuide/html/postinstall.html for the relevant version.

Reply:
Ah yes it's `GEANT4_DATA_DIR` and not `GEANT4_INSTALL_DATADIR`.